contracts/src/integrations/cert/libraries/EigenDACertVerificationLib.sol

```diff
@@ -172,12 +179,22 @@ library EigenDACertVerificationLib {
         // Thus, dividing by codingRate below will only panic if codingRate of a proper initialized version is 0,
         // which is either a configuration bug, or a malicious attack. In both cases, we cannot tell whether the
         // cert is valid or invalid, so it is ok to panic and let social consensus intervene (put a human debugger in the loop).
-        uint256 gamma = securityThresholds.confirmationThreshold - securityThresholds.adversaryThreshold;
-        uint256 n = (10000 - ((1_000_000 / gamma) / uint256(blobParams.codingRate))) * uint256(blobParams.numChunks);
-        uint256 minRequired = blobParams.maxNumOperators * 10000;
 
-        if (!(n >= minRequired)) {
-            revert SecurityAssumptionsNotMet(gamma, n, minRequired);
+        // old code
+        // uint256 gamma = securityThresholds.confirmationThreshold - securityThresholds.adversaryThreshold;
+        // uint256 n = (10000 - ((1_000_000 / gamma) / uint256(blobParams.codingRate))) * uint256(blobParams.numChunks);
+        // uint256 minRequired = blobParams.maxNumOperators * 10000;
+        uint256 lhs = blobParams.codingRate * (blobParams.numChunks - blobParams.maxNumOperators) * (securityThresholds.confirmationThreshold - securityThresholds.adversaryThreshold);
```

Comment:
this could still underflow if `maxNumOperators > numChunks` so we should add that as a separate check